shared/common-adapters/zoomable-box.android.tsx

```diff
@@ -66,53 +75,40 @@ export function ZoomableBox(props: Props) {
 
     // we only activate pan handler when the image is zoomed or user is not pinching
     const pan = Gesture.Pan()
-      .onStart(() => {
-        if (isPinching.value || !isZoomed.value) return
-
+      .averageTouches(true)
```

Comment:
we want the center of touches and not jump around depending on how fingers go up / down